addons/binding/org.openhab.binding.hyperion/README.md

```diff
@@ -0,0 +1,70 @@
+# Hyperion Binding
+
+This binding integrates the Hyperion software with openHAB.
```

Comment:
You could add a link or some more description on what the "Hyperion software" is - I have never heard of it and even by googling, I am having a hard time to find out about it (assuming it is not a financial Oracle product)...